mountpoint-s3/src/inode.rs

```diff
@@ -878,6 +892,14 @@ impl SuperblockInner {
                     });
                 }
 
+                trace!(
+                    same_kind,
+                    same_etag,
+                    existing_is_remote,
+                    remote_is_dir = remote.kind == InodeKind::Directory,
+                    "inode could not be updated in place",
+                );
```

Comment:
I had a weird bug that was difficult to understand, but this trace made it much easier. (Very clear ETag had changed.)